packages/fx-core/resource/package.nls.json

```diff
@@ -111,6 +111,17 @@
   "plugins.spfx.import.success": "Your SPFx solution has been successfully imported to %s.",
   "plugins.spfx.import.log.success": "Teams Toolkit has imported your SPFx solution successfully. A complete log of import details can be found in %s.",
   "plugins.spfx.import.log.fail": "Teams Toolkit failed to import your SPFx solution. A complete log of import details can be found in %s.",
+  "plugins.spfx.addWebPart.confirmInstall": "Teams Toolkit detected your SPFx solution is %s, but you don't have this version installed on your machine. Would you like to install SPFx %s in Teams Toolkit directory to continue adding web part?",
```

Comment:
SPFx version in your solution is %s, not yet installed on your machine. Do you want to install SPFx %s in Teams Toolkit directory to continue adding web part?